lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1415,6 +1416,19 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :description => 'Configures the TCP/IP port for the trace observer Host'
         },
         # Instrumentation
+        :'instrumentation.active_support_notifications.active_support_events' => {
+          :default => NewRelic::Agent::Instrumentation::ActiveSupportSubscriber::EVENT_NAME_TO_METHOD_NAME.keys,
+          :public => true,
+          :type => Array,
+          :allowed_from_server => false,
+          :description => <<~ACTIVE_SUPPORT_EVENTS.chomp.tr("\n", ' ')
+            An allowlist array of Active Support notifications events specific to the Active Support library
+            itself that the agent should subscribe to. The Active Support library specific events focus primarily
+            on caching. Any event name not included in this list will be ignored by the agent. Provide complete event
+            names such as 'cache_fetch_hit.active_support'. Do not provide asterisks or regex patterns, and do not
+            escape any characters with backslashes.
```

Comment:
Reading this brought me back to the Rails guides on instrumentation. It looks like the anchor links aren't quite working, but I'd like to incorporate a link to the full list or list the events we support somewhere in the description.

https://guides.rubyonrails.org/active_support_instrumentation.html 

I also found it interesting that the documentation separates the notifications into "caching" and "messaging" notifications for active support.

Reply:
Great insight. I really want this link to work! Here's a PR that fixes the problem: https://github.com/rails/rails/pull/51696

Reply:
I've added links (that may need to change) with [6a92c5d](https://github.com/newrelic/newrelic-ruby-agent/pull/2610/commits/6a92c5d7238fe0316081a74c8d02dcdaf37f8682)